src/Context/StreamApp.js

```diff
@@ -24,11 +23,13 @@ export type AppCtx<UserData> = {|
   // will stay the same all the time. Because of this react won't notice that
   // the internal fields changed so it thinks it doesn't need to rerender.
   userData: ?UserData,
-  changedUserData: () => void,
   changeNotificationCounts?: any,
   analyticsClient?: any,
   sharedFeedManagers: { [string]: FeedManager },
   errorHandler: ErrorHandler,
+  appId: string | number,
+  apiKey: string,
+  token: string,
```

Comment:
No need to store these items separately in the state. They are part of the client as, `appId`, `apiKey` and `userToken`